setup.cfg

```diff
@@ -34,6 +34,7 @@ python_files = test_*.py
 testpaths = tests
 markers =
 	fuzzing: Run Hypothesis fuzz test suite (deselect with '-m "not fuzzing"')
+	transient: Mark tests using transient storage for differentiation based on EVM version
```

Comment:
this seems like it needs to be updated .. not sure why tests are not failing otherwise

Reply:
do we want to enable `--strict-markers`?

Reply:
yes, probably, i have an open PR which touches setup.cfg though so we should wait until that is merged to avoid conflicts